src/Proto3/Suite/DotProto/Generate/Swagger.hs

```diff
@@ -70,9 +72,9 @@ ppSchema = LC8.putStrLn . encodePretty . toSchema
 -- | JSONPB schemas for protobuf enumerations
 instance (Finite e, Named e) => ToSchema (Enumerated e) where
   declareNamedSchema _ = do
-    let enumName        = nameOf (Proxy @e)
+    let enumName        = nameOf (proxy# :: Proxy# e)
```

Comment:
BTW, one trick is to replace `(proxy# :: Proxy# a)` witih `proxy# @a`.

Reply:
@j6carey type applications don't work with unboxed proxy for some reason:

```
src/Proto3/Suite/DotProto/Generate/Swagger.hs:75:35: error:
    • Couldn't match kind ‘e1’ with ‘*’
      ‘e1’ is a rigid type variable bound by
        the instance declaration
        at src/Proto3/Suite/DotProto/Generate/Swagger.hs:73:10-55
      When matching types
        a0 :: e
        a1 :: *
      Expected type: Proxy# a1
        Actual type: Proxy# a0
    • In the first argument of ‘nameOf’, namely ‘(proxy# @e)’
      In the expression: nameOf (proxy# @e)
      In an equation for ‘enumName’: enumName = nameOf (proxy# @e)
    • Relevant bindings include
        declareNamedSchema :: proxy (Enumerated e1)
                              -> Data.Swagger.Declare.Declare (Definitions Schema) NamedSchema
          (bound at src/Proto3/Suite/DotProto/Generate/Swagger.hs:74:3)
   |
75 |     let enumName        = nameOf (proxy# @e)
   |                                   ^^^^^^^^^
```

Reply:
Oh, right; the first type variable is probably the kind, so you would need to do `proxy# @_ @e` to go this route.

Reply:
As we discussed offline, I'm going to keep the existing code as type ascription is less confusing to read than the double type application version.

Reply:
That's fine.